### ISSUE TITLE
Is this project alive ?

### ISSUE THREAD

**Issue by alpha-and-omega** [bianster/mysqlfs#2]
I just wanted to find fuse to map filesystem to Mariadb galera cluster and replace distributed filesystem with mariadb cluster.
Can I use mysqlfs for this purpose, do fsync->commit work as intended ?

**Comment by bianster** [comment]
I'm not sure. I suggest you try it out with MariaDB and update the thread if you can.

**Comment by loveJesus** [comment]
Hallelujah, i hope to first of all encourage us to seek Jesus Christ http://audacitymovie.com
I just tested it with Galera. I had to change the database from MyISAM to InnoDB in schema.sql and compiling it required me to disable -Werror in the makefile, and make one edit in the code to comment out a print function with a missing constant. 
It is working in that i wrote a file on one machine, it reads it on another. I have not yet stress tested it to any degree. If it works well i plan to share the simple changes mentioned God willing and if i live. Above all i  hope i can encourage us to seek Jesus as He is in the Bible, He does miracles as He always has, and i am thankful to Him for this neat project.
